app/containers/Pages/Table/StrippedTable.js

```diff
@@ -8,58 +10,63 @@ import TableCell from '@mui/material/TableCell';
 import TableHead from '@mui/material/TableHead';
 import TableRow from '@mui/material/TableRow';
 import useStyles from 'enl-components/Tables/tableStyle-jss';
+import { AuthContext } from '../../../AuthContext';
 
-let id = 0;
-function createData(name, calories, fat, carbs, protein) {
-  id += 1;
-  return {
-    id,
-    name,
-    calories,
-    fat,
-    carbs,
-    protein
-  };
-}
+function StrippedTable({ userId }) {
+  const { accessToken } = useContext(AuthContext);
+  const [profileData, setProfileData] = useState({});
+
+  useEffect(() => {
+    const fetchUserData = async () => {
+      try {
+        const res = await fetch('http://localhost:3000/profile', {
```

Comment:
It's good practice to set these type of paths on a `.env` file (set it up as an environment variable), this way, you don't expose the API endpoints on your code, as well as having the leverage of being able to set it up to different APIs depending on where you are running the code (ex.: running locally versus a deployed build on netlify let's say).

If you have this code as it is, and deploy on Netlify, your Netlify FE will not be able to interact with your backend if it tries to reach for localhost. You want to instead set up the environment variable properly through the Netlify settings, and point it to your backend deployed instance.

Reply:
This was an oversight on my part.